### PR TITLE
adding example for old foscam and similar no-brand

### PR DESCRIPTION
**Pull request by yarix**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @yarix, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#23917).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/mjpeg.markdown [hunk 1]
```diff
@@ -56,3 +56,8 @@ use is automatically detected when using a username and password.
 - OctoPrint (OctoPi):
   - MJPEG URL: `http://IP/webcam/?action=stream`
   - Still Image URL: `http://IP/webcam/?action=snapshot`
+
+- Legacy Foscam / wanscam
+  - MJPEG URL: `http://192.168.1.13:8080/videostream.cgi` (add ?resultion=32 for 640x480 or ?resultion=32 for 320x240)
```
Comment (klaasnicolaas) [inline source/_integrations/mjpeg.markdown]:
```suggestion
  - MJPEG URL: `http://IP:PORT/videostream.cgi` (add ?resultion=32 for 640x480 or ?resultion=32 for 320x240)
```

File: source/_integrations/mjpeg.markdown [hunk 2]
```diff
@@ -56,3 +56,8 @@ use is automatically detected when using a username and password.
...
+  - Still Image URL: `http://192.168.1.13:8080/snapshot.cgi`
```
Comment (klaasnicolaas) [inline source/_integrations/mjpeg.markdown]:
```suggestion
  - Still Image URL: `http://IP:PORT/snapshot.cgi`
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]
Thnx for the PR! 🎖️

**Pull request merged**